### PR TITLE
Problem when executing plugin of type command on file level

### PR DESCRIPTION
**Pull request by 47Balboa**

### CODE REVIEW

File: roda-core/roda-core/src/main/java/org/roda/core/plugins/base/conversion/AbstractConvertPlugin.java [hunk 1]
```diff
@@ -473,14 +642,182 @@ protected Report executeOnRepresentation(IndexService index, ModelService model,
...
+                    if (!createDIP) {
+                      try {
+                        Representation rep = model.retrieveRepresentation(file1.getAipId(), newRepresentationID);
+                        createPremisSkeletonOnRepresentation(model, file1.getAipId(), rep);
+                      } catch (RequestNotValidException | GenericException | NotFoundException
+                        | AuthorizationDeniedException | ValidationException | IOException e) {
+                        LOGGER.error("Error running premis skeleton on new representation: {}", e.getMessage());
+                      }
+                    }
+                  }
```
Comment (006627) [inline roda-core/roda-core/src/main/java/org/roda/core/plugins/base/conversion/AbstractConvertPlugin.java:820]: Call the "processFileShallow" method here.

File: roda-core/roda-core/src/main/java/org/roda/core/plugins/base/conversion/AbstractConvertPlugin.java [hunk 2]
```diff
@@ -259,15 +259,183 @@ protected Report executeOnAIP(IndexService index, ModelService model, StorageSer
...
+                        if (!createDIP) {
+                          try {
+                            Representation rep = model.retrieveRepresentation(file1.getAipId(), newRepresentationID);
+                            createPremisSkeletonOnRepresentation(model, file1.getAipId(), rep);
+                          } catch (RequestNotValidException | GenericException | NotFoundException
+                            | AuthorizationDeniedException | ValidationException | IOException e) {
+                            LOGGER.error("Error running premis skeleton on new representation: {}", e.getMessage());
+                          }
+                        }
+                      }
```
Comment (006627) [inline roda-core/roda-core/src/main/java/org/roda/core/plugins/base/conversion/AbstractConvertPlugin.java:438]: Please, wrap this code on a method called "processShallowFile"

**Review by 006627 — CHANGES_REQUESTED** [review]

File: roda-core/roda-core/src/main/java/org/roda/core/plugins/base/conversion/AbstractConvertPlugin.java (hunk 1 above)
Comment (47Balboa) [reply roda-core/roda-core/src/main/java/org/roda/core/plugins/base/conversion/AbstractConvertPlugin.java]: Done

File: roda-core/roda-core/src/main/java/org/roda/core/plugins/base/conversion/AbstractConvertPlugin.java (hunk 2 above)
Comment (47Balboa) [reply roda-core/roda-core/src/main/java/org/roda/core/plugins/base/conversion/AbstractConvertPlugin.java]: Done

**Pull request merged**